src/lib/routes.js

```diff
@@ -3,13 +3,19 @@
 const dateFormat = 'YYYY-MM-DD HH:mm:ss';
 const moment = require('moment');
 const model = require('./model');
-const report = require('lev-react-components').LevReport;
+const dashboardModel = require('./dashboard-model');
+const { LevDashboard, LevReport } = require('lev-react-components');
 
-module.exports = server => {
-  server.get('/readiness', (req, res) => {
-    res.send('OK');
+const promiseResponder = (promise, component, req, res, next) => promise
```

Comment:
Could the component go at the end to make calling the function prettier when you want to send json?

Reply:
yep good call - fixed